Makefile

```
@@ -659,7 +659,8 @@ examples/examples.gen.json: scripts/examplegen/main.go examples/examples.go $(sh
 	go run ./scripts/examplegen/main.go > examples/examples.gen.json
 
 coderd/rbac/object_gen.go: scripts/typegen/rbacobject.gotmpl scripts/typegen/main.go coderd/rbac/object.go coderd/rbac/policy/policy.go
-	go run scripts/typegen/main.go rbac object > coderd/rbac/object_gen.go
+	go run scripts/typegen/main.go rbac object > /tmp/object_gen.go
+	mv /tmp/object_gen.go coderd/rbac/object_gen.go
```

Made it a little more verbose and writing to a temp dir instead 

```suggestion
	tempdir=$(shell mktemp -d /tmp/typegen_rbac_object.XXXXXX)
	go run ./scripts/typegen/main.go rbac object > "$$tempdir/object_gen.go"
	mv -v "$$tempdir/object_gen.go" coderd/rbac/object_gen.go
	rmdir -v "$$tempdir"
```

---

Sweet! We should do this, or something similar for all generated files. Each one is done ad hoc iirc.